ux.symfony.com/src/Twig/ProductGrid.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Twig;
+
+use App\Service\EmojiCollection;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveAction;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\ComponentToolsTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent('ProductGrid')]
+class ProductGrid
+{
+    use ComponentToolsTrait;
+    use DefaultActionTrait;
+
+    private const PER_PAGE = 10;
+
+    #[LiveProp]
+    public int $page = 1;
+
+    public function __construct(private readonly EmojiCollection $emojis)
+    {
+    }
+
+    #[LiveAction]
+    public function more(): void
```

Comment:
```suggestion
    public function next(): void
```

?

Reply:
Not sure, as the idea is there to push the "load more" idea, instead of a "next page" ... maybe loadMore() ?